The target "GetTargetPath" does not exist in the project

Hi @AArnott, 
I am using this tool in a solution that contains a SqlServer project.
When building the database project, I receive the following error.

##[error]packages\nerdbank.gitversioning\3.5.109\build\PrivateP2PCaching.proj(0,0): Error MSB4057: The target "GetTargetPath" does not exist in the project.

If I modify the file `PrivateP2PCaching.proj` by adding the following

```
  <!-- database project wants to see this target -->
  <Target Name="GetTargetPath" />
```
The solution builds successfully.

I am not sure of the implication of this change.

The following are the logs for the failing build using [msbuild structured log](https://msbuildlog.com/).

[msbuild.binlog.zip](https://github.com/dotnet/Nerdbank.GitVersioning/files/9609625/msbuild.zip)

Is this something you can look into?

I really appreciate any help you can provide.

### Update
I was using version [3.5.109](https://www.nuget.org/packages/Nerdbank.GitVersioning/3.5.109). If I downgrade the version to [3.4.255](https://www.nuget.org/packages/Nerdbank.GitVersioning/3.4.255), where the file `PrivateP2PCaching.proj` is not part of the referenced projects, then the build process ends successfully.

--- a/src/Nerdbank.GitVersioning.Tasks/build/PrivateP2PCaching.proj
+++ b/src/Nerdbank.GitVersioning.Tasks/build/PrivateP2PCaching.proj
@@ -22,15 +22,18 @@
   <Target Name="SatelliteDllsProjectOutputGroup" />
   <Target Name="BuiltProjectOutputGroupDependencies" />
   <Target Name="DebugSymbolsProjectOutputGroup" />
-  
+
   <!-- WinUI3 seems to want this target -->
   <Target Name="GetTargetPathWithTargetPlatformMoniker" />
-  
+
   <!-- dotnet watch needs this target -->
   <Target Name="_CollectWatchItems" />
-  
+
   <!-- VS Test wants to see this target -->
   <Target Name="VSTest" />
+
+  <!-- SqlServer projects want this target-->
+  <Target Name="GetTargetPath" />
 
   <Import Project="$(NBGV_PrivateP2PAuxTargets)" Condition=" '$(NBGV_PrivateP2PAuxTargets)' != '' " />
 </Project>

